dev/ci/ci-basic-overlay.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# This is the basic overlay set for repositories in the CI.
+
+# Maybe we should just use Ruby to have real objects...
```

Comment:
I would argue against using Ruby so that one can build the CI targets locally without requiring a new dependency.